src/css/categories.css

```
@@ -116,16 +115,17 @@
 
   .jobs-categories-wrapper {
     gap: 25px;
+    max-width: 860px;
   }
 
   .category-card {
     text-align: center;
   }
 }
 
-@media (max-width: 1023px) {
-  .jobs-categories-wrapper {
-    max-width: 565px;
+@media (max-width: 940px) {
+  .category-card {
+    width: 48%;
```

```suggestion
    width: 50%;
```
This means the card will use 50% of the available space: `100% of the container - gap width / 2`